### DISCUSSION TITLE
S3 upload asset management

### DISCUSSION THREAD

**Discussion by tksham** [mozilla/hubs#5523] [Hubs Cloud Questions] [answered]
I tried deleting the assets that I uploaded via my own spoke scene editor,
however I can still go to the direct link and get the assets.
e.g. https://boaxmeta-1-assets.boaxmeta.com/files/bfe4ff26-f99a-4eae-bc58-49db28e7a8fa.jpg

Is there anyway to delete the file permanently? I am not sure if it still uses my S3 storage space for cost.
And is there any way to backup everything on S3? 

Sorry I am new to AWS. it would be very kind of someone to point me to a direction or guide me with keywords for me to search. Thanks!

**Comment by pattersonbl2** [comment, marked as answer]
It would still be related to storage cost but it's very little.  With S3, you pay for data and the access to date but it's not a lot. 

You should be able to use AWS Backup to back up S3, Also you can use S3 glacier service to save cost and back up old files.

**Reply by tksham** [reply]
sometimes my models are >100MB, how would it be very little? and how if I really want to clean them properly?
